### DISCUSSION TITLE
deb release schedule?

### DISCUSSION THREAD

**Discussion by wtip** [raspberrypi/picamera2#726] [Q&A]
How would one determine when a release here on github gets built and pushed to the deb repository?

**Comment by wtip** [comment]
[Release 0.3.10](https://github.com/raspberrypi/picamera2/releases/tag/v0.3.10) was published 3 weeks ago and still isn't available in the apt repository.

**Reply by KLRico** [reply]
I'm not sure how the updates get pushed to that repo either. I was eager to try out the multiple encoder support, so I ended up removing the apt package and installing with pip (which has the recent releases).

**Comment by wtip** [comment]
Interestingly Release 0.3.12 came out today and was also published in the deb repo.

**Reply by KLRico** [reply]
Hmm... I was under the assumption that the push to deb was automated, but I guess not.

Well now that it's current, I wonder if I should switch back to installing/updating with apt, or stay with pip? I know the install guide says doing it through apt is the preferred method, but I'm not sure why that is.

**Reply by tvoverbeek** [reply]
The reason is with apt picamera2 updates are in sync with libcamera updates.
With pip if a libcamera update is needed to support the picamera2 update (and v.v.) you are on your own.
